frontend/src/components/PlayCollection.js

```diff
@@ -9,8 +9,8 @@ export default function PlayCollection({ plays }) {
   <>
   <h1>Plays</h1>
     <StyledPlayCollection>
-      {plays && plays.map((play, index) => (
-        <Play play={play} key={index} />
+      {plays && plays.map((play, idx) => (
+        <Play play={play} key={idx} />
```

Comment:
Ich glaube hier wolltest du noch genauere IDs vergeben? Da sich die Indizes ja ändern könnten.

Reply:
Ja, das habe ich gemacht, und zwar uuids. Nach dem Aufsetzen des Servers und Speichern des Eintrags in der Datenbank wurden diese obsolet.